cmd/pipeline/deploy.go

```diff
@@ -127,16 +128,28 @@ func deploy(ctx context.Context) *cobra.Command {
 	cmd.Flags().BoolVarP(&wait, "wait", "w", false, "wait until the pipeline finishes (defaults to false)")
 	cmd.Flags().BoolVarP(&skipIfExists, "skip-if-exists", "", false, "skip the pipeline deployment if the pipeline already exists in the namespace (defaults to false)")
 	cmd.Flags().DurationVarP(&timeout, "timeout", "t", (5 * time.Minute), "the length of time to wait for completion, zero means never. Any other values should contain a corresponding time unit e.g. 1s, 2m, 3h ")
+	cmd.Flags().StringArrayVarP(&variables, "var", "v", []string{}, "set a pipeline variable (can be used multiple times)")
 	return cmd
 }
 
-func deployPipeline(ctx context.Context, name, namespace, repository, branch string, wait bool, timeout time.Duration) error {
+func deployPipeline(ctx context.Context, name, namespace, repository, branch string, wait bool, timeout time.Duration, variables []string) error {
 	spinner := utils.NewSpinner("Deploying your pipeline...")
 	spinner.Start()
 	defer spinner.Stop()
 
+	varList := []okteto.Variable{}
+	for _, v := range variables {
+		kv := strings.SplitN(v, "=", 2)
+		if len(kv) != 2 {
```

Comment:
would it make sense to support flag-like variables?

Reply:
What would be the value of the variable? `true` or a behavior similar to `-e`  on docker compose. https://docs.docker.com/compose/environment-variables/#pass-environment-variables-to-containers

`The value of the DEBUG variable in the container is taken from the value for the same variable in the shell in which Compose is run.`